importer/main.py

```diff
@@ -256,12 +256,14 @@ def main(
                     del _["code"]
                     issues.append(_)
                 logging.error(json_response)
+            logging.info(inventory_creation_response.text)
+            logging.info("GROUPS: " + str(groups_created))
```

Comment:
Adding these to help with debugging. If you are running an import and the Groups or Lists are empty, then there is obviously an issue somewhere